pkg/compare/compare.go

```diff
@@ -518,11 +583,41 @@ func (obj InfoObject) Name() string {
 	return slug.Make(apiKindNamespaceName(obj.clusterObj))
 }
 
+func (obj InfoObject) getPatch() (string, error) {
+	localRuntime, err := obj.Merged()
```

Comment:
i am not sure if the patch should created from the version that already contains the patched version (like still not  sure about the case if there is a load and save patch at the same time  what will each of them include)

i would of expect here to use the object only with the ommited fields (without the load patch)

Reply:
Yeah good point now that the other PR is merged I'll have to update this one :)